xonsh/pyghooks.py

```diff
@@ -1388,7 +1388,10 @@ def color_file(file_path: str, mode: int) -> (Color, str):
         except FileNotFoundError:
             color_key = "or"
     elif stat.S_ISREG(mode):
-        if stat.S_IMODE(mode) & (stat.S_IXUSR + stat.S_IXGRP + stat.S_IXOTH):
+        cap = os.listxattr(file_path)
```

Comment:
I don't believe so.  If only I could get CI to run, it would try Windows for us...

Reply:
I mean, on other platforms it will raise an AttributeError, module 'os' has no attribute 'listxattr', in the same way that e.g. the Windows-specific os.startfile raises on linux...

Reply:
You are correct.  And Windows files don't have capabilities anyway.

Reply:
Exactly. The same is true for other OSes (BSD, etc.).